Modifying getting started tutorial to match GE 101 webinar content (#1828)
great-expectations/great_expectations: docs/guides/tutorials/getting_started/initialize_a_data_context.rst

.. _tutorials__getting_started__initialize_a_data_context:

> Set up the tutorial data and initialize a Data Context

Reviewer: Please make analogous change in the name of the step in the Getting Started document to maintain parallelism. (currently, it still reads "Initialize a DataContext")

(small nit as well: Sphinx produces a warning if the length of the line of header-delimiters (e.g. '=' in this case) is less than the header itself).

===============================================

> In Great Expectations, your :ref:`Data Context` manages your project configuration. Using a Data Context is almost always the fastest way to get up and running, even though some teams don't need every component of a Data Context.

Reviewer: I strongly prefer the new "your project" wording, and would make the same change in the overview description of this step in the getting started (overview) doc.

Author: Good catch, I had changed that in the first iteration of this PR but then didn't go back. And yes, I'm trying to simplify/clarify the language a little.

    pip install great_expectations

If you intend to develop within Great Expectations (e.g. to contribute back to the project), and would like to install it from a git branch or a fork, check out :ref:`contributing_setting_up_your_dev_environment` in the contributor documentation.

Preparation instructions
---------------------

For this tutorial, we will use a simplified version of the NYC taxi ride data. We prepared a Docker image that contains a local Postgres database with the data pre-loaded, so you can easily get up and running without any local dependencies.

To avoid confusion during the tutorial, we recommend you follow these steps:

> #. Make sure you have `Docker<https://www.docker.com/>`_ installed

Reviewer: Broken link (probably just needs a space before the '<')

![image](https://user-images.githubusercontent.com/332575/90848757-a3dd8200-e33b-11ea-812d-a9b9b187de37.png)

Author: The rst struggle is real.

> #. Clone the ``ge_tutorials`` repository and start up the container with the Postgres database containing the data:

Reviewer: Turn ge_tutorials into an external link?

Author: Good idea!

.. code-block:: bash

   git clone https://github.com/superconductive/ge_tutorials
   cd ge_tutorials/ge_getting_started_tutorial
   docker-compose up

> You will now have a Postgres database running with some pre-loaded data! In case you're looking to connect to the database, you'll find instructions in the ``README`` in the repository.

Reviewer: ```suggestion
You will now have a Postgres database running with some pre-loaded data! In case you're looking to connect to the database, you'll find instructions in the `README <https://github.com/superconductive/ge_tutorials>`_ in the repository.
```

Author: Changed the link to point directly to the right subdirectory.

About the data
-----------------------------------------------

The NYC taxi data is an open data set which is updated every month. Each record corresponds to one taxi ride and contains information such as the pick up and drop-off location, the payment amount, and the number of passengers, among others.

> In this tutorial, we provide two tables with a 10,000 row sample each of the taxi data:

Reviewer: ```suggestion
In this tutorial, we provide two tables, each with a 10,000 row sample of the taxi data:
```

Author: 👍

* **yellow_tripdata_sample_2019_01**: a sample of the January 2019 taxi data
* **yellow_tripdata_staging**: a sample of the February 2019 taxi data, loaded to a "staging" table so we can validate it before promoting it to a permanent table

If we compare the ``passenger_count`` column in the January and February data, we find a significant difference: The February data contains a large proportion of rides with 0 passengers, which seems unexpected.

.. admonition:: The data problem we're solving in this tutorial

>     In this tutorial, we will be creating an Expectation Suite for this example data set that allows us to assert that we generally expect at least 1 passenger per taxi ride based on what we see in the January 2019 data. We will then use that Expectation Suite to catch the data quality issue in the February 2019 staging data.

Reviewer: <3 <3 <3

So clear and to the point.

    OK to proceed? [Y/n]: 

**Let's pause there for a moment and take a look under the hood.**

The ``great_expectations/`` directory structure
-----------------------------------------------

> Once you finish going through ``init``, your ``great_expectations/`` directory will contains all of the important components of a Great Expectations deployment, in miniature:

Reviewer: ```suggestion
Once you finish going through ``init``, your ``great_expectations/`` directory will contain all of the important components of a Great Expectations deployment:
```

Author: Changed this and added "local" deployment, for consistency. As an aside, I'm making changes on my local branch, so not accepting them through github.
